src/widgets/main.lisp

```diff
@@ -63,11 +63,11 @@
    (make-sponsors-widget))
   ("/versions/\\d+"
    (make-version-widget))
-  ("/projects/.*/.*"
+  ("/projects/(?!.*\\.svg$).*/.*"
    (make-project-widget))
   ("/dists/.*"
    (make-dist-widget))
-  ("/projects/.*"
+  ("/projects/(?!.*\\.svg$).*"
```

Comment:
This seems wrong, because that way we are changing route of the make-author-projects-widget.
And few lines abover we are breaking route for the make-project-widget.

A separate route for badge-svg should be added here.